ci-operator/step-registry/hypershift/mce/dump/hypershift-mce-dump-commands.sh

```diff
@@ -31,15 +31,16 @@ else
 fi
 
 CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
+HOSTED_CLUSTER_NS=$(oc get hostedcluster -A -ojsonpath='{.items[0].metadata.namespace}')
```

Comment:
In the test, we only deploy one HC isn't it?

Reply:
yes, We're doing this just to obtain the namespace of the hosted cluster, as the namespace used for ibmz is not local-cluster